pkg/network/vmicontroller/status.go

```diff
@@ -51,6 +51,16 @@ func UpdateStatus(clusterConfig clusterConfigChecker, vmi *v1.VirtualMachineInst
 
 	interfaceStatuses = append(interfaceStatuses, secondaryIfaceStatuses...)
 
+	// virt-handler adds status for interfaces only existing inside the guest that were discovered by QEMU GuestAgent
+	guestInternalIfaceStatuses := vmispec.FilterInterfaceStatuses(
+		vmi.Status.Interfaces,
+		func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool {
+			return ifaceStatus.Name == "" && vmispec.ContainsInfoSource(ifaceStatus.InfoSource, vmispec.InfoSourceGuestAgent)
+		},
+	)
```

Comment:
Please consider export this to function as other examples in this context (calc. secondary/primary ifaces..) 
I could make the code less bulky and easier to follow, having descriptive name can eliminate the need for commenting what the code does.

Reply:
Done.